src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -578,43 +572,7 @@ export default function SelectNextPage() {
             component.
           </p>
 
-          <Library.Code
-            content={`function App() {
-  const [value, setSelected] = useState<{ id: string; name: string }>();
-  return (
-    <SelectNext
-      value={value}
-      onChange={setSelected}
-      buttonContent={
-        value ? (
-          <>
-            {value.name}
-            <div className="rounded px-2 ml-2 bg-grey-7 text-white">
-              {value.id}
-            </div>
-          </>
-        ) : (
-          <>Select one…</>
-        )
-      }
-    >
-      {items.map(item => (
-        <SelectNext.Option value={item} key={item.id}>
-          {() => (
-            <>
-              {item.name}
-              <div className="grow" />
-              <div className="rounded px-2 bg-grey-7 text-white">
-                {item.id}
-              </div>
-            </>
-          )}
-        </SelectNext.Option>
-      ))}
-    </SelectNext>
-  );
-}`}
-          />
+          <Library.Code exampleFile="select-next-main" />
```

Comment:
The basic "how do I use this" example needs to be at the top of the page, with a demo/source toggle.

Reply:
I have converted the first example (the basic one) into one using this new example-file approach, and removed this example at the bottom entirely.

It was only a workaround due to the lack of code snippets for the other existing examples, so it's no longer necessary anyway.